pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -233,43 +254,86 @@ func (m *manager) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAdmitR
 	klog.Infof("[topologymanager] Topology Admit Handler")
 	pod := attrs.Pod
 
-	for _, container := range append(pod.Spec.InitContainers, pod.Spec.Containers...) {
-		if m.policy.Name() == PolicyNone {
+	// Exception - Policy : none
+	if m.policy.Name() == PolicyNone {
+		for _, container := range append(pod.Spec.InitContainers, pod.Spec.Containers...) {
 			err := m.allocateAlignedResources(pod, &container)
 			if err != nil {
-				return lifecycle.PodAdmitResult{
-					Message: fmt.Sprintf("Allocate failed due to %v, which is unexpected", err),
-					Reason:  "UnexpectedAdmissionError",
-					Admit:   false,
-				}
+				return podAdmitError(unexpectedAdmissionError, err)
 			}
-			continue
 		}
+		return lifecycle.PodAdmitResult{Admit: true}
+	}
+	klog.Infof("[topologymanager] Topology Scope : %v", m.topologyScope)
+	if m.topologyScope == podScopeTopology {
+		return m.podBasisAdmit(pod)
```

Comment:
nit naming again - will the `Scope` term be better than `Basis` here? Just to avoid introducing new terms in code

Reply:
right, I get rid of `Basis`. `podAdmit` and `containerAdmit` are self explanatory